app/src/main/res/layout/home_dialog.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/dialogRecyclerView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" >
+
+    </androidx.recyclerview.widget.RecyclerView>
+
+    <Spinner
+        android:id="@+id/homeSpinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/grey_border"
```

Comment:
This makes it impossible to see that it actually is a spinner. Are there other styles for spinners available (maybe in the material components library)?

Reply:
Oh yea that was just to distinguish it, could be changed, any ideas?

Reply:
I would do an "Outlined exposed dropdown". See https://github.com/material-components/material-components-android/blob/7939c171dffbe2687d3e12bf00a973d79a839d27/docs/components/Menu.md#exposed-dropdown-menu-example

<img width="250" src="https://user-images.githubusercontent.com/5811634/120068931-7e786400-c083-11eb-9711-b52de5702a9a.png" />

Reply:
done